k8s/zookeeper/README.md

```diff
@@ -149,22 +150,22 @@ for i in "IMAGE_ZOOKEEPER" "IMAGE_ZOOKEEPER_EXPORTER" "IMAGE_METRICS_EXPORTER" ;
 done
 ```
 
-Define amount of replicas for ZooKeeper:
+Define the amount of replicas for ZooKeeper:
 
-> **NOTE:** This number should be odd for normal work of ZooKeeper.
+> **NOTE:** This number should be odd for typical configurations of ZooKeeper.
```

Comment:
It's not about typical configuration. Zookeeper service requires a majority to run, so it is best practice to use odd number of machines. Even number is supported, but is not advised because it will require more peers for a quorum.

Reply:
Thanks for the clarification on this! I gave rephrasing it a shot - let me know what you think! Thanks again for the thoughtful review.

Reply:
Great, thanks :)